installation/scripts/run.sh

```diff
@@ -4,6 +4,7 @@ ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
 
 KYMA_RELEASE="1.1.0"
 COMPASS_HELM_RELEASE_NAME="compass"
+COMPASS_HELM_RELEASE_NAMESPACE="kyma-system"
```

Comment:
Consider installing compass to different namespace than `kyma-system`, maybe just `compass`?